RELEASE-NOTES.md

```diff
@@ -1,12 +1,33 @@
 # Riak CS 2.0.0-pre1 Release Notes
 
-- based upon Riak 2.0
+- First release that works with Riak 2.0
 - configuration format moved line-by-line style
 
-## Changes
+## Changes and Additions
 
 - Changed default value of `gc_max_workers` from 5 to 2 with its name
   changed to `gc.max_workers` with migration to config format change.
+- Partial support of GET Location API (#1057)
+- Add AWS v4 header authentication (#1064) - note that there are a lot
```

Comment:
Some additional words needed, partial or preliminary or half-cooked or ...
I want to stress "it is not for production yet."